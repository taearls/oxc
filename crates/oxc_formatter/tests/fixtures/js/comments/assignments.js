var longlonglonglonglonglong = /*#__PURE__*/_interopDefaultLegacy(aaaaaaaaaaaaaaa);
var short = /*#__PURE__*/_interopDefaultLegacy(b);

const jestPackageJson =
  // eslint-disable-next-line @typescript-eslint/no-require-imports
  require(jestPath);
