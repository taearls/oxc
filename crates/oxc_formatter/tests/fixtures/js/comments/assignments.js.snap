---
source: crates/oxc_formatter/tests/fixtures/mod.rs
---
==================== Input ====================
var longlonglonglonglonglong = /*#__PURE__*/_interopDefaultLegacy(aaaaaaaaaaaaaaa);
var short = /*#__PURE__*/_interopDefaultLegacy(b);

const jestPackageJson =
  // eslint-disable-next-line @typescript-eslint/no-require-imports
  require(jestPath);

==================== Output ====================
var longlonglonglonglonglong =
  /*#__PURE__*/ _interopDefaultLegacy(aaaaaaaaaaaaaaa);
var short = /*#__PURE__*/ _interopDefaultLegacy(b);

const jestPackageJson =
  // eslint-disable-next-line @typescript-eslint/no-require-imports
  require(jestPath);

===================== End =====================
